feature/experimental/gribi/otg_tests/backup_nhg_action/backup_nhg_action_test.go

```diff
@@ -467,6 +496,11 @@ func createFlow(t *testing.T, ate *ondatra.ATEDevice, top gosnappi.Config, name
 	innerIPHeader := flow.Packet().Add().Ipv4()
 	innerIPHeader.Src().SetValue(innerSrcIP1)
 	innerIPHeader.Dst().Increment().SetStart(innerDstIP1).SetCount(1)
+	flow.EgressPacket().Add().Ethernet()
+	ipTracking := flow.EgressPacket().Add().Ipv4()
+	ipDst := ipTracking.Dst().MetricTags().Add()
+	ipDst.SetName("ipDstTracking").SetOffset(0).SetLength(12)
+
```

Comment:
The second procedure does require checking for source IP in one case, can we add that for this case here?

https://github.com/openconfig/featureprofiles/tree/main/feature/experimental/gribi/otg_tests/backup_nhg_action#procedure

Reply:
@jasdeep-hundal : Custom offset is not supported yet, so I have validated src ip and dst ip separately. Please check if that works.

Reply:
Validating src/dst separately is preferred because it's more explicit (and it's represented easily in the disaggregated OTG flow stats as well.)